specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json

```diff
@@ -2530,13 +2534,22 @@
           "enum": [
             "Standard_LRS",
             "Standard_ZRS",
-            "Premium_LRS"
+            "Premium_LRS",
+            "PremiumV2_LRS"
           ],
           "x-ms-enum": {
             "name": "StorageAccountType",
             "modelAsString": true
           }
         },
+        "additionalReplicaSets": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AdditionalReplicaSet"
+          },
+          "x-ms-identifiers": [],
+          "description": "List of storage sku with replica count to create direct drive replicas."
+        },
         "encryption": {
```

Comment:
I didn't realize there was more on this list. 
Let's put "additionalReplicaSets" at the end of TargetRegion.properties list. 

That is to preserve the parameters list order for TargetRegion constructors.